src/API/Router.ts

```diff
@@ -90,6 +95,31 @@ export class Router {
     context.body = work
   }
 
+  private getHealth = async (context: KoaRouter.IRouterContext, next: () => Promise<any>) => {
+    const {
+      mongoIsConnected,
+      ipfsIsConnected,
+      blockchainInfo,
+      walletInfo,
+      networkInfo,
+      ipfsDownloadRetries,
+      hardIpfsFailures,
+      ipfsFailureWillRetry,
+      ipfsFailureMaxRetries,
+    } = await this.healthController.getHealth()
```

Comment:
Isn't is simpler to just `context.body = await this.healthController.getHealth()`?